app/src/main/java/org/simple/clinic/summary/PatientSummaryInit.kt

```diff
@@ -36,4 +30,18 @@ class PatientSummaryInit : Init<PatientSummaryModel, PatientSummaryEffect> {
 
     return first(model, effects)
   }
+
+  private fun checkTeleconsultationInfo(
+      model: PatientSummaryModel,
+      effects: MutableSet<PatientSummaryEffect>
+  ) {
+    when (model.teleconsultInfo) {
+      null, is TeleconsultInfo.Fetching -> {
```

Comment:
If we are in the `Fetching` state, why would we want to fetch again?

Reply:
Since we are checking during screen restore state, the model may not be updated from the initial request. So, just to be safe I also considering `Fetching` state.